regolith/helpers/u_contacthelper.py

```diff
@@ -75,9 +75,10 @@ def db_updater(self):
         name = HumanName(rc.name)
         frag = fragment_retrieval(self.gtx['contacts'], ["_id", "aka", "name"], rc.name)
         ids = set(i.get('_id') for i in frag)
+        sorted_ids = sorted(ids)
         index = 1
         all_names = {}
-        for i in ids:
+        for i in sorted_ids:
             currentid = rc.client.find_one(rc.database, rc.coll, {'_id': i})
```

Comment:
I am not sure what all this code is.  can't you do it with something like this?
```
sorted_ids = list(set(i.get("_id") for n in fragment_retrieval(self.gtx['contacts'], ["_id", "aka", "name"], rc.name))).sort()
index = list(range(len(sorted_ids)))
print(*[f"{i}. {j}\n" for i, j, in zip(index, sorted_ids))
```
if you need other things from the retrieved entries you can assign the return from fragment retrieval to a variable and get the ids from that.

Reply:
ok! done!